devel/boost/files/patch-add-support-for-arm64.diff

```diff
@@ -0,0 +1,56 @@
+--- tools/build/src/tools/darwin.jam.orig
++++ tools/build/src/tools/darwin.jam
+@@ -431,6 +431,7 @@
+     local support-ppc64 = 1 ;
+     
+     osx-version ?= $(.host-osx-version) ;
++    local osx-version-split = [ regex.split $(osx-version) \\. ] ;
+ 
+     switch $(osx-version)
+     {
+@@ -440,7 +441,7 @@
+         }
+         
+         case * :
+-        if $(osx-version) && ! [ version.version-less [ regex.split $(osx-version) \\. ] : 10 6 ]
++        if $(osx-version) && ! [ version.version-less $(osx-version-split) : 10 6 ]
+         {
+             # When targeting 10.6:
+             # - gcc 4.2 will give a compiler errir if ppc64 compilation is requested
+@@ -452,7 +453,16 @@
+     {
+         case combined : 
+         {
+-            if $(address-model) = 32_64 {
++            if ! [ version.version-less $(osx-version-split) : 11 0 ]
++            {
++                # macOS 11.0 "Big Sur" and later is always 64-bit ...
```

Comment:
The 10.14 SDK and later is always 64-bit...

Reply:
true ... but that's not what this check is for. It is purely for whether the macOS version is "not < 11.0". My goal in this PR is to fix the build for ARM64, which in turn means macOS 11.0.

I can easily add in some logic below with the line starting `} else if $(address-model) = 32_64 {` to handle the case you note ... but that would be in a separate PR IMHO.